### DISCUSSION TITLE
Unable to get single language

### DISCUSSION THREAD

**Discussion by devgene** [spatie/laravel-translatable#298] [Q&A]
Unable to get single language translation in paginate collection. I need to get json response with the help of language header and pagination

**Comment by RomkaLTU** [comment]
I don't know if there is a better way, but the only way I manage to do it is by overriding toArray method in model:

```
public function toArray(): array
{
    ...
    'name' => $this->name,
    ...
}
```

after that call toArray() on paginate()

`paginate()->toArray()`
